dvc/exceptions.py

```diff
@@ -47,7 +47,8 @@ def __init__(self, output, stages):
         )
         msg = (
             "file/directory '{}' is specified as an output in more than one "
-            "stage: {}"
+            "DVC-file, which is not allowed: {}\n"
+            "Consider using a different output name."
         ).format(output, "\n    ".join(stages))
```

Comment:
Maybe use back ticks with something like

```suggestion
         ).format(output, '`' + "`\n,  `".join(stages) + '`')
```

? (Probably not that exact code.)

Reply:
We use single ticks inline and no quoting on separate lines everywhere else, so I would leave it as is.

Reply:
True